public/app/core/history/RichHistoryStorage.ts

```diff
@@ -50,3 +51,19 @@ export default interface RichHistoryStorage {
   getSettings(): Promise<RichHistorySettings>;
   updateSettings(settings: RichHistorySettings): Promise<void>;
 }
+
+/**
+ * Simple data source srv mock used for testing remote and local storage conversions between
+ * uid and data source name.
+ */
+export const DataSourceSrvMock = {
```

Comment:
is this the right place for this? 🤔

Reply:
Good point. Actually it's not really needed, might be easier/better to test it with real DataSourceSrv - 77e2336482ac06c5a63e4923b80dfb50d0153932